### PR TITLE
Changes ACF to native composer install

### PR DESCRIPTION
**Pull request by JPrevost**
NOTE: terminus secret `ACF_PRO_KEY` should be removed only after _all_ environments have been updated with this change. It will cause no harm to keep it there, but it will not actually do anything once these changes land and should be removed.

Why are these changes being introduced:

* ACF now allows direct install and the shim we used is being retired

Relevant ticket(s):

* https://mitlibraries.atlassian.net/browse/LM-236

How does this address that need:

* Removes acf-composer-bridge repository
* Adds acf native repository

Document any side effects to this change:

`COMPOSER_AUTH` env is required in local .env as well as being set via terminus secret plugin.

### Secrets

- [x] All new secrets have been added to Pantheon tiers
- [ ] Relevant secrets have been updated in Github Actions
- [x] All new secrets documented in README

### Documentation

- [ ] Project documentation has been updated
- [ ] No documentation changes are needed

### Accessibility

- [ ] ANDI or Wave has been run in accordance to
      [our guide](https://mitlibraries.github.io/guides/basics/a11y.html) and
      all issues introduced by these changes have been resolved or opened as new
      issues (link to those issues in the Pull Request details above)

### Stakeholder approval

- [ ] Stakeholder approval has been confirmed
- [x] Stakeholder approval is not needed

### Dependencies

YES dependencies are updated (updated to new version of ACF during this process to address security concern)

## Code Reviewer

- [ ] The commit message is clear and follows our guidelines
      (not just this pull request message)
- [ ] The changes have been verified
- [ ] The documentation has been updated or is unnecessary
- [ ] New dependencies are appropriate or there were no changes

### CODE REVIEW

**Review by matt-bernhardt — COMMENTED** [review]
This appears to work now locally, but I have a question about what I'm seeing in GH Actions. 

The local steps I took were:

* Add an `auth.json` to my repo root based on what's in LastPass
* Run `composer install` and watch the new version of ACF get installed. Success!

After this, I cleaned out the `ACF_PRO_KEY` entry in my `.env` file, and ran a few other commands - which had the expected output. `composer outdated` is one that, I think, ends up checking all of the various packages against their declared source to determine what is outdated - which should also allow us to confirm that we have the needed access.

---
When I look at GH Actions, it looks like those are now failing in a way that isn't related to this change - there's a message that `composer.lock` is outdated? Is that expected, or something that we need to change?

On a maybe-related note, and not blocking this change, I want to ask your thoughts on whether we need to clean up LastPass - it looks like there's two folders now, and one of them has what looks like an outdated `.env` file. I'm happy to wait until we see Adam's documentation suggestions based on his first experiences here, though.

Either way, this works for me locally - which is very promising. My only "blocking" question is the one about whether GH Actions should be in the state it is.

**Comment by JPrevost** [comment]
@matt-bernhardt I was holding off on looking at Actions until we confirmed we were okay with this approach as fixing it in this branch will break every other branch but I will make sure what you are seeing is resolved along with the change in the variable syntax. I will work on fixing Actions now for this branch now that we have both seen success in Pantheon and locally. I won't merge until I tag you again for review, but this is the point at which every other branch will fail CI until we merge this branch (I mean, we can always change the ENV back to the old way if this takes too long and we need some other code to land but... yeah).

As for LastPass... yeah, we're still in transition between the old and new LastPass folder structure. I'm not entirely sure why that is being held up at the DLS level but I suspect everyone (like us) just hasn't had time to complete our transition to the new structure.

**Comment by JPrevost** [comment]
@matt-bernhardt this should be ready for final sign off as the tests are passing after the Actions ENV were updated unless you see anything else we should talk through before we merge

**Review by matt-bernhardt — APPROVED** [review]
Looks good! I'm excited about getting this done. Thanks a bunch :-)

:shipit:

**Pull request merged**
